Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.3)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    json (2.3.0)
-    jwt (2.2.1)
-    jwt_signed_request (2.5.4)
+    json (2.3.1)
+    jwt (2.2.2)
```

Comment:
### [jwt](https://github.com/jwt/ruby-jwt)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.2.1 → 2.2.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **22** commits. See the full changes on [the compare page](https://github.com/jwt/ruby-jwt/compare/v2.2.1...v2.2.2).

These are the first 10 commits:
- (ecf3f67) [Removed forwardable dependency](https://github.com/jwt/ruby-jwt/commit/ecf3f67b3fefa59de339df397edc60c8a5fa7dc9)
- (0b21e60) [Add rails 6.rc1 gemfile](https://github.com/jwt/ruby-jwt/commit/0b21e60df3c351c1ade2be14067a6459f441c707)
- (9d05f30) [Make travis to run appraisal introduced in #278](https://github.com/jwt/ruby-jwt/commit/9d05f305d2e7224afc51dc0aec3581e3aceb466f)
- (d49906f) [Exclude expected failures](https://github.com/jwt/ruby-jwt/commit/d49906f2dc51bd225b106fcd3083c59ff591a475)
- (cb38890) [Merge pull request #326 from berkos/add_latest_rails_version](https://github.com/jwt/ruby-jwt/commit/cb3889021cf582675d24ad5d73cb49d2f3f83c4e)
- (a269358) [Merge pull request #325 from anakinj/remove-forwardable-depe…](https://github.com/jwt/ruby-jwt/commit/a269358ab782a46f24f792257f25ef4e72489b2e)
- (a9c0aa8) [Support RSA.import for all Ruby versions.](https://github.com/jwt/ruby-jwt/commit/a9c0aa8abaa1d8c92a03a2ea76f60dd5e2367ac7)
- (f4c7f15) [Merge pull request #333 from rahulbajaj0509/import-ruby-issu…](https://github.com/jwt/ruby-jwt/commit/f4c7f15ccce3e74a475ddd9931eb7ccc5b594818)
- (ab4b881) [Pin simplecov to < 0.18](https://github.com/jwt/ruby-jwt/commit/ab4b88173c56d67bae5947b8763903bdd0d6a3c3)
- (12fef19) [Merge pull request #356 from anakinj/try-to-fix-ci](https://github.com/jwt/ruby-jwt/commit/12fef19a343d568aeb3c428bae5735cfbd022696)


</details>